test/e2e/networking.go

```diff
@@ -121,8 +121,19 @@ var _ = Describe("Networking", func() {
 		filterNodes(nodes, func(node api.Node) bool {
 			return isNodeReadySetAsExpected(&node, true)
 		})
+		if len(nodes.Items) < 1 {
+			Failf("No Ready nodes found.")
+		}
+		// in general, the test requires two nodes. But for local development, often a one node cluster
```

Comment:
nit: maybe move this comment above the `if len nodes.Items < ` block?  very minor, but in case your refactoring might make more sense.